docker/models/model.py

```diff
@@ -110,7 +110,7 @@ def receiver():
 
     # receive 1 message at a time, call callback function
     channel.basic_qos(prefetch_count=1)
-    channel.basic_consume(callback, queue=rabbitmq_queue)
+    channel.basic_consume(on_message_callback=callback, queue=rabbitmq_queue)
```

Comment:
This same change also needs to be in the `executor.py` file:

https://github.com/PecanProject/pecan/blob/810f4f0f601e76ef6e06c9df494dba7faba863b6/docker/executor/executor.py#L113